docs/USAGE-GUIDE.md

```diff
@@ -1113,6 +1113,10 @@ connection. **s2n_recv** will return the number of bytes read and also return
 
 **NOTE:** Unlike OpenSSL, repeated calls to **s2n_recv** should not duplicate the original parameters, but should update **buf** and **size** per the indication of size read. For example;
 
+**NOTE:** By default, **s2n_recv** will return after reading a single TLS record. To change this
+behavior such that it will read up to **size**, the config for the connection can be updated
+by calling **s2n_config_enable_recv_multi_record**.
```

Comment:
You inserted the new note between the old note and its example

Reply:
wasn't sure if there was a precedent for multiple notes :)

how would you like this formatted?

Reply:
I'd probably put it before the existing note. Putting it after the example will probably make everyone miss it. But it definitely can't got between "For example:" and the example :)